core/src/main/java/com/spotify/ffwd/output/CoreOutputManager.java

```diff
@@ -181,6 +200,34 @@ public final Long getHLLPRefreshPeriodLimit() {
         } else {
             this.cardinalityLimit = null;
         }
+
+        if (!Strings.isNullOrEmpty(dynamicTagsFile)) {
+            final File file = new File(dynamicTagsFile);
+            if (file.exists() && file.canRead()) {
+                final int initialDelay =
+                    System.getenv().containsKey("IS_FFWD_CONFIGURATION_TEST") ? 3 : 60;
+                tagRefreshingThread = Executors.newSingleThreadScheduledExecutor(
+                    new ThreadFactoryBuilder().setNameFormat("tag-refreshing-thread").build());
+                tagRefreshingThread.scheduleWithFixedDelay(() -> refreshTagsFromFile(file.toPath()),
+                    initialDelay, 60, TimeUnit.SECONDS);
+            } else {
+                log.warn("Cannot read file at {}. Ignoring it.", dynamicTagsFile);
+            }
+        }
+    }
+
+    private void refreshTagsFromFile(final Path path) {
+        try (final Stream<String> lines = Files.lines(path)) {
+            lines.forEach(label -> {
+                final String[] labelParts = label.split("=", 2);
+                // Only update tag values if the tag keys are explicitly specified in normal configs
+                if (tags.get(labelParts[0]) != null) {
+                    tags.put(labelParts[0], labelParts[1]);
+                }
+            });
+        } catch (IOException e) {
+            // Ignore
```

Comment:
I was thinking about this case and I kinda think we should exit here with error - the main idea that this code will be executed only in case user indicated the tags are coming via labels.yaml file and if file either corrupted or doesn't exist the ffwd should exit as it does when configPath is not set or null. 
What you think?

Reply:
@malish8632 -- @lmuhlha and I spoke with @davidxia yesterday; plan to ignore is because the dynamic tags file will be exclusive to the new/ACA tags _for now_, leaving all pre-existing tags configured as they currently are. Moving forward, we can re-evaluate as to whether or not the file becomes the single source for tags, at which point we would handle this Exception differently.

Reply:
For tags that are known to be static for the lifetime of the process, I think it's fine to continue using env vars. So you can limit the use of this functionality to this use case: 1) you have tags that are known to change 2) and you can't restart the process with updated env vars.

Reply:
Makes sense. Thanks